src/track/track_gps_l1ca.c

```diff
@@ -337,11 +344,20 @@ static void tracker_gps_l1ca_update(const tracker_channel_info_t *channel_info,
 
     /* Indicate that a mode change has occurred. */
     common_data->mode_change_count = common_data->update_count;
+
+    do_l1ca_to_l2cm_handover(common_data->sample_count,
```

Comment:
The issue of L2CM re-acquisition should be addressed by another PR https://github.com/swift-nav/piksi_firmware/pull/681. In that PR it indeed tries to re-acquire L2CM periodically unless it is already in track.